spec/spec_helper.rb

```diff
@@ -1,8 +1,8 @@
 require "codeclimate-test-reporter"
 require "coveralls"
-# require "simplecov"
-Coveralls.wear!
-# SimpleCov.start
+ require "simplecov"
+#Coveralls.wear!
```

Comment:
Incorrect indentation detected (column 0 instead of 1).<br>Missing space after #.